pkg/apis/workflow/v1alpha1/types.go

```diff
@@ -133,6 +133,8 @@ type WorkflowSpec struct {
 	// allowed to run before the controller terminates the workflow. A value of zero is used to
 	// terminate a Running workflow
 	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty"`
+	// Priority is used if controller is configured to process limited number of workflows in parallel. Workflows with higher priority are processed first.
+	Priority int `json:"priority,omitempty"`
```

Comment:
We should keep this consistent with the PodSpec and make this an int32 pointer:
```
Priority *int32 `json:"priority,omitempty" protobuf:"bytes,25,opt,name=priority"`
```

The difference between zero priority vs. omitted priority I think is important. For example, we may decide to support something like a default priority in the configmap, which at that point we have no way of knowing the intent.

Reply:
Good point. Fixed